acquisitions/src/main/resources/thunderjet/mod-orders/features/item-and-holding-operations-for-order-flows.feature

```diff
@@ -0,0 +1,211 @@
+@parallel=false
+# for https://issues.folio.org/browse/MODORDERS-538
+Feature: check Items and holding process.
+
+  Background:
+    * url baseUrl
+    #* callonce dev {tenant: 'testorders1'}
+    * callonce loginAdmin testAdmin
+    * def okapitokenAdmin = okapitoken
+    * callonce loginRegularUser testUser
+    * def okapitokenUser = okapitoken
+    * def headersAdmin = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenAdmin)', 'Accept': 'application/json'  }
+    * def headersUser = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenUser)', 'Accept': '*/*'  }
+    * configure headers = headersUser
+
+    * callonce variables
+
+    * def fundId = callonce uuid1
+    * def budgetId = callonce uuid2
+    * def orderId = callonce uuid3
+    * def poLineId = callonce uuid4
+    * def initialInstanceId = globalInstanceId1
+    * def initialHoldingId = globalHoldingId1
+
+
+  Scenario: Create finances
+    # this is needed for instance if a previous test does a rollover which changes the global fund
+    * configure headers = headersAdmin
+    * call createFund { 'id': '#(fundId)'}
+    * call createBudget { 'id': '#(budgetId)', 'allocated': 10000, 'fundId': '#(fundId)'}
+
+  Scenario: Create an order
+    * configure headers = headersUser
+    Given path 'orders/composite-orders'
+    And request
+    """
+    {
+      id: '#(orderId)',
+      vendor: '#(globalVendorId)',
+      orderType: 'One-Time'
+    }
+    """
+    When method POST
+    Then status 201
+
+  Scenario: Create an order line
+    * def poLine = read('classpath:samples/mod-orders/orderLines/simple-order-line.json')
+    * set poLine.id = poLineId
+    * set poLine.purchaseOrderId = orderId
+    * set poLine.instanceId = initialInstanceId
+    * set poLine.isPackage = false
+    * set poLine.fundDistribution[0].fundId = fundId
+    * remove poLine.locations[0].locationId
+    * set poLine.locations[0].holdingId = initialHoldingId
+    Given path 'orders/order-lines'
+    * configure headers = headersUser
+    And request poLine
+    When method POST
+    Then status 201
+    * def createdLine = $
+    * def poLineNumber = createdLine.createdLine
```

Comment:
Seems this line should be corrected, it is not po line number, also it is not used, so we can delete

Reply:
deleted